spec/chromium-spec.js

```diff
@@ -998,55 +997,64 @@ describe('chromium feature', () => {
   })
 
   describe('PDF Viewer', () => {
-    const pdfSource = url.format({
-      pathname: path.join(fixtures, 'assets', 'cat.pdf').replace(/\\/g, '/'),
-      protocol: 'file',
-      slashes: true
-    })
-    const pdfSourceWithParams = url.format({
-      pathname: path.join(fixtures, 'assets', 'cat.pdf').replace(/\\/g, '/'),
-      query: {
-        a: 1,
-        b: 2
-      },
-      protocol: 'file',
-      slashes: true
-    })
-
-    function createBrowserWindow ({plugins, preload}) {
-      w = new BrowserWindow({
-        show: false,
-        webPreferences: {
-          preload: path.join(fixtures, 'module', preload),
-          plugins: plugins
-        }
-      })
-    }
+    before(function () {
+      if (!features.isPDFViewerEnabled()) {
+        return this.skip()
+      }
+    })
 
-    function testPDFIsLoadedInSubFrame (page, preloadFile, done) {
-      const pagePath = url.format({
-        pathname: path.join(fixtures, 'pages', page).replace(/\\/g, '/'),
+    beforeEach(() => {
+      this.pdfSource = url.format({
```

Comment:
`this === global` here I think 😨 

Can we avoid `this` and just used scoped `let` one scope higher 👍

Reply:
> `this === global` here I think 

Nope ) It's kinda a test execution context, it's totally find to store there anything.
And it's a good way to conditionally define/create things in `before/beforeEach` hooks.

Reply:
Arrow functions aren't bound to mochas context correctly (they don't create their own `this` scope, they will always inherit the one above and can't be bound with `.bind`)

Reply:
It means that you shouldn't use arrow functions is you want a new context to be created.